tools/f18/flang.sh.in

```diff
@@ -26,4 +26,4 @@ function abspath() {
 
 wd=`abspath $(dirname "$0")/..`
 
-${wd}/bin/f18 -module-suffix .f18.mod -intrinsic-module-directory ${wd}/include $*
+${wd}/bin/f18 -module-suffix .f18.mod -intrinsic-module-directory @FLANG_INTRINSIC_MODULES_DIR@ $*
```

Comment:
Why is the intrinsic module directory a cmake variable substitution but the path to f18 still relative to this script? Shouldn't they both be found using a similar mechanism?

Reply:
The f18 binary will always be in build_dir/bin/f18, whereas the location of the modules is different if you're in an in-tree or out-of-tree build. Strictly speaking you're probably correct, but the f18 path will be correct for both builds here so it doesn't really matter.